oct/ansible/oct/playbooks/package/ami-mark-ready.yml

```diff
@@ -55,4 +55,13 @@
         resource: '{{ origin_ci_aws_ami_id }}'
         tags:
           ready: 'yes'
-        state: present
\ No newline at end of file
+        state: present
+
+    - name: add any additional tags
```

Comment:
> but is evident that if you use it you won't actually package it in that same run.

I can add the tag added by --mark-ready in the same dict I include the rest of the tags and make the tagging atomic but if you feel strong about keeping it separate I am fine with it.